### PR TITLE
PCA unit test

### PR DESCRIPTION
**Pull request by snova-petrojm**
- Added unit tests for each of the 8 analysis in post call analysis (PCA) kit in tests/pca_test.py
- Added a tests/test_config.yaml file
- Changed do_sample to False in src/analysis.py (to make answer deterministic)
- Updated test_framework.py

### CODE REVIEW

**Review by snova-kwasia — CHANGES_REQUESTED** [review]
Great work @snova-petrojm ! My change is super minor around maintainability.

File: post_call_analysis/config.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ llm:
 embedding_model: 
     "type": "sambastudio" # set either sambastudio or cpu
     "batch_size": 1 #set depending of your endpoint configuration (1 if CoE embedding expert)
-    "coe": True #set true if using Sambastudio embeddings in a CoE endpoint 
+    "coe": False #set true if using Sambastudio embeddings in a CoE endpoint
```
[the remark this answers is not in the document]
Comment (snova-petrojm) [reply post_call_analysis/config.yaml]: done, thanks

File: post_call_analysis/tests/pca_test.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+logger.info(f'kit_dir: {kit_dir}')
+logger.info(f'repo_dir: {repo_dir}')
+
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
+
+from post_call_analysis.src import analysis, plot, asr
+
+audio_save_location=(os.path.join(kit_dir,"data/conversations/audio"))
```
[the remark this answers is not in the document]
Comment (snova-petrojm) [reply post_call_analysis/tests/pca_test.py]: done, thanks

**Review by snova-jorgep — APPROVED** [review]

**Comment by snova-kwasia** [comment]
Looks good thanks @snova-petrojm!

**Pull request merged**